homeassistant/components/fritz/services.yaml

```diff
@@ -0,0 +1,13 @@
+reconnect:
+  description: Reconnects your FRITZ!Box internet connection
+  target:
+    entity:
+      integration: fritz
+      domain: binary_sensor
+
+reboot:
+  description: Reboots your FRITZ!Box
+  target:
+    entity:
+      integration: fritz
+      domain: binary_sensor
```

Comment:
If the goal is to target the binary sensor entities, register these are entity services

https://developers.home-assistant.io/docs/dev_101_services/#entity-services

Reply:
The idea was to target only a device, but didn't find a way to filter out all entities from GUI.
So I choice to filter by binary sensor and allow both to work ( device and entity).